mindmeld/components/question_answerer.py

```diff
@@ -455,6 +458,8 @@ def filter(self, **kwargs):
         For example, in food ordering domain the resolved restaurant entity can be used as a filter
         to resolve dish entities. The exact knowledge base field to apply these filters depends on
         the knowledge base data model of the application.
+        If the entity is not in the canonical form, a fuzzy filter can be applied by setting the
```

Comment:
is there an example of the use case of adding fuzzy filters?

Reply:
In the case of Cisco product documents, the title could be `catalyst 3000` but the query can have `cat 3k`. So if we wanted to filter by matching against titles, we'd need a character n-gram match.